### ISSUE TITLE
[padding-line-between-statements] Documented `cjs-import` and `cjs-export` statement types are not accepted

### ISSUE THREAD

**Issue by gebsh** [eslint-stylistic/eslint-stylistic#53]
[The documentation](https://eslint.style/rules/default/padding-line-between-statements#rule-details) of the `padding-line-between-statements` rule states that `cjs-import` and `cjs-export` are valid statement types.

However, linting the following JS file with the following command results in an error.

```js
const path = require('node:path');
module.exports = {};
```

```console
gebsh@gebsh % eslint --no-eslintrc --env es2023 --plugin '@stylistic' --rule '@stylistic/padding-line-between-statements: [error, { blankLine: always, prev: cjs-import, next: cjs-export }]' ./test.cjs

Oops! Something went wrong! :(

ESLint: 8.53.0

Error: CLIOptions:
	Configuration for rule "@stylistic/padding-line-between-statements" is invalid:
	Value "cjs-import" should be equal to one of the allowed values.
	Value "cjs-import" should be array.
	Value "cjs-import" should match some schema in anyOf.
```

I believe this is caused by the fact that `@typescript-eslint` renames these two options to `require` and `exports`: https://github.com/typescript-eslint/typescript-eslint/issues/4091. Renaming them in the configuration does work:

```console
gebsh@gebsh % eslint --no-eslintrc --env es2023 --plugin '@stylistic' --rule '@stylistic/padding-line-between-statements: [error, { blankLine: always, prev: require, next: exports }]' ./test.cjs

<project-root>/test.cjs
  2:1  error  Expected blank line before this statement  @stylistic/padding-line-between-statements

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```
